arangod/Sharding/ShardingFeature.cpp

```diff
@@ -91,9 +93,10 @@ void ShardingFeature::prepare() {
   // note: these standins will actually not do any sharding, but always
   // throw an exception telling the user that the selected sharding
   // strategy is only available in the Enterprise Edition
-  for (auto const& name : std::vector<std::string>{
-           "enterprise-smart-edge-compat", "enterprise-hash-smart-edge"}) {
-    registerFactory(name, [name](ShardingInfo* sharding) {
+  for (auto const& name : std::array<std::string, 3>{
+           "enterprise-smart-edge-compat", "enterprise-hash-smart-edge",
+           "enterprise-hex-smart-vertex"}) {
```

Comment:
hex was missed?

Reply:
I don't understand what "hex" you are referring to here. Can you please elaborate?

Reply:
"enterprise-hex-smart-vertex" wasn't in this list before your changes

Reply:
Ah ok, yes. When that sharding strategy was added, it wasn't mentioned here.
Mentioning it in this place of the code isn't critical, but will provide a better error message to the end user in case they are using that sharding strategy (which is only available in enterprise edition) with a community edition build. That can only happen by using a hot backup or database directory copy from an enterprise edition deployment and running community edition against it.